netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -322,6 +322,16 @@ public ConnectionClientTransport newClientTransport(
       }
       ProtocolNegotiator negotiator = protocolNegotiator != null ? protocolNegotiator :
           createProtocolNegotiator(authority, negotiationType, sslContext);
+      String proxy = System.getenv("GRPC_PROXY_EXP");
```

Comment:
Calling it a _convention_ is probably giving it too much credit. I would have explained what it meant, but I instead used the words to describe it being deleted immediately (and thus, it shouldn't matter).